avm/utilities/pipelines/platform/Assign-Issue.ps1

```diff
@@ -0,0 +1,99 @@
+Function Get-AvmCsv {
```

Comment:
Misses documentation

Reply:
@rahalan you can source the Get-AvmCSV from the avm repo - this has docs and all functions are seperate modules

Reply:
Done